kartothek/io_components/docs.py

```diff
@@ -184,7 +184,7 @@ def default_docs(func):
                         whitespaces = " " * whitespaces_to_add
                         doc = whitespaces + doc
                         doc = doc.replace("\n", "\n" + whitespaces).rstrip() + "\n"
-                        if whitespaces + param not in docs:
+                        if doc not in docs:
```

Comment:
Since this is highly meta, can you also add a comment explaining what this is doing?